src/ui/QuoteMessage/utils.ts

```diff
@@ -0,0 +1,73 @@
+import {CoreMessageType, isFileMessage, isMultipleFilesMessage} from "../../utils";
+import {match} from "ts-pattern";
+import {FileMessage, MultipleFilesMessage, UploadedFileInfo} from "@sendbird/chat/message";
+import {Thumbnail} from "@sendbird/chat/lib/__definition";
+
+export function getMessageFirstFileType(message: CoreMessageType): string {
+  return match(message)
+    .when(m => isFileMessage(m), () => {
+      return (message as FileMessage)?.type ?? '';
+    })
+    .when(m => isMultipleFilesMessage(m), () => {
+      const castedMessage: MultipleFilesMessage = message as MultipleFilesMessage;
+      return getFirstFileInfo(castedMessage)?.mimeType ?? '';
+    })
+    .otherwise(() => {
+      return '';
+    });
+}
+
+function getFirstFileInfo(message: MultipleFilesMessage): UploadedFileInfo {
+  const fileInfoList: UploadedFileInfo[] = message.fileInfoList;
+  return fileInfoList.length > 0 ? fileInfoList[0] : null;
+}
+
+export function getMessageFirstFileName(message: CoreMessageType): string {
+  return match(message)
+    .when(m => isFileMessage(m), () => {
+      const castedMessage: FileMessage = message as FileMessage;
+      return castedMessage?.name ?? '';
+    })
+    .when(m => isMultipleFilesMessage(m), () => {
+      const castedMessage: MultipleFilesMessage = message as MultipleFilesMessage;
+      return getFirstFileInfo(castedMessage)?.fileName ?? '';
```

Comment:
```suggestion
      return (message as FileMessage)?.name ?? '';
    })
    .when(isMultipleFilesMessage, () => {
      return (message as MultipleFilesMessage)?.fileName ?? '';
```

Like you did at the above in `getMessageFirstFileType`,  we can make it simpler in that way. 
Same for `getMessageFirstFileUrl` or `getMessageFirstFileThumbnails`.

Reply:
ah nice suggestion! castedMessage is no longer needed after replacing repeating code with `getFirstFileInfo`.